completions.py

```diff
@@ -722,7 +722,7 @@ def on_query_completions(self, view, prefix, locations):
         #   e.g. force:appHostable, force:lightningQuickAction
         if ch in ['"', ',', ' ']:
             matched_region = view.find('(implements="[\\w\\s:\\,]+")', begin)
-            if matched_region and matched_region.contains(pt): 
+            if matched_region and pt < matched_region.b:
```

Comment:
It failed to pass validation, @lushang

Reply:
Let me check it again. any better way to fix this issue?

Reply:
Let it go firstly, if you have better solution, please pull.